### PR TITLE
[Keyboard] Add mini_ten_key_plus

### PR DESCRIPTION
**Pull request by minibois**
## Description
Add mini_ten_key_plus keyboard/numpad.
Wasn't sure if this 5x5 layout (default keymap) was fit for the "ortho_5x5" default layout.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* N/A

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/mini_ten_key_plus/readme.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# mini_ten_key_plus
+
+![mini_ten_key_plus](imgur.com/QtFFONg.jpg)
```
Comment (drashna) [inline keyboards/mini_ten_key_plus/readme.md:3]:
```suggestion
![mini_ten_key_plus](imgur.com/QtFFONgl.jpg)
```

File: keyboards/mini_ten_key_plus/readme.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+*A short description of the keyboard/project*
```
Comment (drashna) [inline keyboards/mini_ten_key_plus/readme.md:5]: fill this out or remove it. 
```suggestion
```

File: keyboards/mini_ten_key_plus/rules.mk [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
```
Comment (drashna) [inline keyboards/mini_ten_key_plus/rules.mk:17]:
```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/mini_ten_key_plus/readme.md (hunk 2 above)
Comment (minibois) [reply keyboards/mini_ten_key_plus/readme.md]: Missed that one, I've since added a description and accepted the other two changes.

**Review by drashna — APPROVED** [review]

File: keyboards/mini_ten_key_plus/rules.mk [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+NKRO_ENABLE = no            # USB Nkey Rollover
```
Comment (fauxpark) [inline keyboards/mini_ten_key_plus/rules.mk:15]:
```suggestion
NKRO_ENABLE = no            # Enable N-Key Rollover
```

File: keyboards/mini_ten_key_plus/readme.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
+# mini_ten_key_plus
+
+![mini_ten_key_plus](imgur.com/QtFFONgl.jpg)
```
Comment (fauxpark) [inline keyboards/mini_ten_key_plus/readme.md:3]:
```suggestion
![mini_ten_key_plus](https://imgur.com/QtFFONgl.jpg)
```

File: keyboards/mini_ten_key_plus/keymaps/standard_numpad/readme.md [hunk 6]
```diff
@@ -0,0 +1,2 @@
+# The standard numpad keymap for mini_ten_key_plus
+![mini_ten_key_plus](imgur.com/x2BUc9J.png)
```
Comment (fauxpark) [inline keyboards/mini_ten_key_plus/keymaps/standard_numpad/readme.md:2]:
```suggestion
![mini_ten_key_plus](https://imgur.com/x2BUc9J.png)
```

File: keyboards/mini_ten_key_plus/keymaps/default/readme.md [hunk 7]
```diff
@@ -0,0 +1,2 @@
+# The default 1u keymap for mini_ten_key_plus
+![mini_ten_key_plus](imgur.com/VuSTwfC.png)
```
Comment (fauxpark) [inline keyboards/mini_ten_key_plus/keymaps/default/readme.md:2]:
```suggestion
![mini_ten_key_plus](https://imgur.com/VuSTwfC.png)
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**